label_studio/projects/functions/__init__.py

```diff
@@ -22,19 +22,13 @@ def annotate_finished_task_number(queryset):
 
 def annotate_total_predictions_number(queryset):
     if flag_set(
-        'fflag_fix_back_lsdv_4719_improve_performance_of_project_annotations',
+        'fflag_perf_back_lsdv_4695_update_prediction_query_to_use_direct_project_relation',
```

Comment:
Is the fflag_fix_back_lsdv_4719_improve_performance_of_project_annotations still used at all, or is this intended to be removed?

Reply:
`fflag_fix_back_lsdv_4719_improve_performance_of_project_annotations` has been serving True to all for 7 months. Safe to remove.